checkov/terraform/checks/resource/azure/AzureContainerInstanceEnvVarSecureValueType.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
```

Comment:
@praveen-panw Let's use Python's built in types wherever possible, i.e `dict` instead of `Dict`, `list` instead of `List` 🙂

Reply:
Okay 👍🏻